homeassistant/components/camera/generic.py

```diff
@@ -40,6 +41,7 @@
     vol.Optional(CONF_PASSWORD): cv.string,
     vol.Optional(CONF_USERNAME): cv.string,
     vol.Optional(CONF_CONTENT_TYPE, default=DEFAULT_CONTENT_TYPE): cv.string,
+    vol.Optional(CONF_FRAMERATE, default=2): cv.positive_int,
```

Comment:
1. This is not a true framerate. Because the I/O always takes some time, the effective frame rate will always be lower.
2. Why is this an integer? Why not a positive float?

Reply:
Thanks for the feedback, regarding #13350 it covers a different use case. 

1. I don't see that as an issue. Consider this the desired maximum framerate. If I the user sets it to 15 and HA only does 13, I don't see that as an issue. Framerate is always dependent on performance. 
2. Why would floats be needed for this?

Reply:
This will always be an approximate frame rate, and configuring a positive_int is more natural